### DISCUSSION TITLE
Running memos on the server, not using docker

### DISCUSSION THREAD

**Discussion by nibazshab** [usememos/memos#2772] [Show and tell]
It's so easy. We just need get `memos` binary file, and `dist` files

1. install docker and pull this image in another machine (or in any other way)
2. run it. and then use `docker ps -a` to find it ID
3. use `docker cp ID:/usr/local/memos/memos ./memos` copy the binary file
4. use `docker cp ID:/usr/local/memos/dist ./dist` copy the dist files ( version >= 0.18.2. I'm not sure whether v0.18.1 need this step, I don't test it. )
5. send files to your server, and run it

```
dist  memos  memos_demo.db  memos_prod.db
-bash-4.2# ./memos -d . -p 5230 -m prod
---
Server profile
data: /opt/memos
dsn: /opt/memos/memos_prod.db
addr: 
port: 5230
mode: prod
driver: sqlite
version: 0.18.2
metric: true
---

███╗   ███╗███████╗███╗   ███╗ ██████╗ ███████╗
████╗ ████║██╔════╝████╗ ████║██╔═══██╗██╔════╝
██╔████╔██║█████╗  ██╔████╔██║██║   ██║███████╗
██║╚██╔╝██║██╔══╝  ██║╚██╔╝██║██║   ██║╚════██║
██║ ╚═╝ ██║███████╗██║ ╚═╝ ██║╚██████╔╝███████║
╚═╝     ╚═╝╚══════╝╚═╝     ╚═╝ ╚═════╝ ╚══════╝
Version 0.18.2 has been started on port 5230
---
See more in:
👉Website: https://usememos.com
👉GitHub: https://github.com/usememos/memos
---
```

**Comment by etjec4** [comment]
what about run it on windows?

**Reply by nibazshab** [reply]
I don't know. :(

**Reply by lincolnthalles** [reply]
[Read this to run the server on Windows](https://github.com/lincolnthalles/memos-builds/blob/main/docs/windows-service.md)

[Check this](https://github.com/lincolnthalles/memospot) to use Memos a stand-alone application on Windows, macOS or Linux.

**Comment by M0ustach3** [comment]
Hi everyone,

I made a PR on NixOS' repository to have a service to run Memos ! If you want to check it out at https://github.com/NixOS/nixpkgs/pull/426687
